app/controllers/blogs_controller.rb

```diff
@@ -48,12 +49,8 @@ def set_blog
   end
 
   def blog_params
-    require_params = params.require(:blog)
-    params_all_user = :title, :content, :secret
-    if current_user.premium?
-      require_params.permit(params_all_user, :random_eyecatch)
-    else
-      require_params.permit(params_all_user)
-    end
+    params_of_blog = :title, :content, :secret
```

Comment:
```suggestion
    params_of_blog = [:title, :content, :secret]
```

Rubyの構文上元のコードのままでも動きますが、`[]`を付けた方がわかりやすいと思いますー

Reply:
[]をつけました。